pkg/kudoctl/cmd/plan/plan_status.go

```diff
@@ -25,8 +27,40 @@ func Status(options *Options, settings *env.Settings) error {
 	return status(kc, options, settings.Namespace)
 }
 
+func statusFormatted(kc *kudo.Client, options *Options, ns string) error {
+	instance, err := kc.GetInstance(options.Instance, ns)
+	if err != nil {
+		return err
+	}
+	if instance == nil {
+		return fmt.Errorf("instance %s/%s does not exist", ns, options.Instance)
+	}
+
+	var o []byte
+	if strings.ToLower(options.Output) == "yaml" {
+		o, err = yaml.Marshal(instance.Status)
```

Comment:
shouldn't we aim for something more user friendly then just serialized CRD status? 🤔

Reply:
That's a good question. I'm not sure what would be more user friendly though...

Reply:
the rest of k8s simply outputs the resource yaml - why would we convert? I'd rather leave it as is

Reply:
> I would like to see a test here :) at least I can see how the output actually look like

@alenkacz Added tests :) And ended up refactoring this quite a bit. The output for yaml/json plan status is still the instance.status though.

It might be a good idea to use a separate data structure so we don't break clients if we end up changing the instance.status....